code/game/machinery/cryopod.dm

```diff
@@ -472,6 +474,28 @@ var/global/list/frozen_items = list("Alpha"=list(),"Bravo"=list(),"Charlie"=list
 			//Despawning occurs when process() is called with an occupant without a client.
 			add_fingerprint(M)
 
+/obj/machinery/cryopod/verb/return_to_lobby()
+	set name = "Return to Lobby"
+	set category = "Object"
+	set src in oview(1)
```

Comment:
```suggestion
	set src in view(0)
```

Reply:
`oview(1)` does a 3x3 scan of the object excluding the center.
`view(0)` only includes the center, which is what we want, for this to appear when the user is in the same tile.